### PR TITLE
fix: green field installation fixed - added missing service

### PR DESCRIPTION
**Pull request by boajer**
https://oat-sa.atlassian.net/browse/CON-185

### CODE REVIEW

File: scripts/install/RegisterAssetTreeBuilder.php [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+namespace oat\taoItems\scripts\install;
+
+use oat\oatbox\extension\InstallAction;
+use oat\tao\model\ClientLibRegistry;
+use oat\tao\model\asset\AssetService;
+
+namespace oat\taoItems\scripts\install;
```
Comment (voknil) [inline scripts/install/RegisterAssetTreeBuilder.php:27]: Duplicated string?

File: scripts/install/RegisterAssetTreeBuilder.php [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA;
+ */
+
+namespace oat\taoItems\scripts\install;
+
+use oat\oatbox\extension\InstallAction;
+use oat\tao\model\ClientLibRegistry;
+use oat\tao\model\asset\AssetService;
```
Comment (shpran) [inline scripts/install/RegisterAssetTreeBuilder.php:25]: As I see, this lines can be dropped
```suggestion
```

**Review by shpran — APPROVED** [review]
Unnecessary lines can be dropped

- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by bartlomiejmarszal — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [x] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by bziondik — APPROVED** [review]
Tested locally, works

**Pull request merged**